invenio_communities/config.py

```diff
@@ -102,6 +102,20 @@
 }
 """Community roles."""
 
+# TODO temporarily added options for UIs - to be moved?
```

Comment:
This should be removed. It is not configurable because it is a boolean `True`/`False` and you cannot have a third value.
To be hardcoded in the React app.